simSPI/linear_simulator/configs/betagal_distribution.cfg

```diff
@@ -0,0 +1,71 @@
+[general]
+    name = betagal_distribution
+            #str: name of the dataset folder
+    usegpu= False
+            #bool: (default: False) option to use GPU TODO: gpu option to be added
+    datasetSize= 100
+            #int: (default: 410) size of the dataset
+    chunks=10
+            #int: (default: 10) size of the chunk per iteration
+
+[paths]
+    input_volume_path = simSPI/linear_simulator/configs/betagal.mrc
```

Comment:
Unless they are used for tests, I am not sure we want to store data in a GitHub repo. The .mrc and .star files should be found somewhere else (see OSF - @geoffwoollard has been looking into that with his students); this would also be an opportunity to describe how these files were generated, where they come from, etc.